{"name": "test_vwo_login_positive", "status": "broken", "statusDetails": {"message": "TypeError: LoginPage.login_to_vwo() got an unexpected keyword argument 'username'", "trace": "setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"004162f32a525a90f3fa4cd045011866\")>\n\n    @allure.epic(\"VWO Login Test\")\n    @allure.feature(\"TC#1 - VWO App Positive Test\")\n    @pytest.mark.positive\n    def test_vwo_login_positive(setup):\n        driver = setup\n        loginPage = LoginPage(driver)\n>       loginPage.login_to_vwo(username=\"contact+atb5x@thetestingacademy.com\", password=\"Wingify@SDET\")\nE       TypeError: LoginPage.login_to_vwo() got an unexpected keyword argument 'username'\n\ntests\\vwoLoginTests\\test_vwo_login.py:41: TypeError"}, "start": 1715679623494, "stop": 1715679623495, "uuid": "8dc11af3-d507-44a2-8b34-3a9e2c534157", "historyId": "99091ff4f3a54bbba7777a96f0a2fe5e", "testCaseId": "99091ff4f3a54bbba7777a96f0a2fe5e", "fullName": "tests.vwoLoginTests.test_vwo_login#test_vwo_login_positive", "labels": [{"name": "feature", "value": "TC#1 - VWO App Positive Test"}, {"name": "epic", "value": "VWO Login Test"}, {"name": "tag", "value": "positive"}, {"name": "parentSuite", "value": "tests.vwoLoginTests"}, {"name": "suite", "value": "test_vwo_login"}, {"name": "host", "value": "DESKTOP-PUBF6P0"}, {"name": "thread", "value": "12272-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwoLoginTests.test_vwo_login"}]}